lua/lspconfig/powershell_es.lua

```diff
@@ -4,20 +4,27 @@ local util = require 'lspconfig/util'
 local server_name = 'powershell_es'
 local temp_path = vim.fn.stdpath 'cache'
 
-local function make_cmd(bundle_path)
-  if bundle_path ~= nil then
+local function make_cmd(new_config)
+  local command = ""
+  if new_config.bundle_path ~= nil then
     local command_fmt =
       [[%s/PowerShellEditorServices/Start-EditorServices.ps1 -BundledModulesPath %s -LogPath %s/powershell_es.log -SessionDetailsPath %s/powershell_es.session.json -FeatureFlags @() -AdditionalModules @() -HostName nvim -HostProfileId 0 -HostVersion 1.0.0 -Stdio -LogLevel Normal]]
-    local command = command_fmt:format(bundle_path, bundle_path, temp_path, temp_path)
-    return { 'pwsh', '-NoLogo', '-NoProfile', '-Command', command }
+    command = command_fmt:format(new_config.bundle_path, new_config.bundle_path, temp_path, temp_path)
   end
+  if new_config.shell == nil then
```

Comment:
Rather than settings new_config.shell == pwsh, just have a default value for shell (pwsh) which is overridden by new_config.shell if it exists.

Reply:
Added `shell` member to `default_config` with the default value of `pwsh`. Is that what you were asking for?